clang/test/CIR/CodeGen/bitfields.c

```diff
@@ -0,0 +1,82 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+
+struct __long {
+  struct __attribute__((__packed__)) {
```

Comment:
Nice testcase, looks like it's coming from a system header!

Reply:
it came from bitfields.cpp  :blush: